pkg/scheduler/internal/queue/scheduling_queue_test.go

```diff
@@ -650,10 +650,12 @@ func TestPriorityQueue_NominatedPodsForNode(t *testing.T) {
 		t.Errorf("Expected: %v after Pop, but got: %v", highPriorityPodInfo.Pod.Name, p.Pod.Name)
 	}
 	expectedList := []*framework.PodInfo{medPriorityPodInfo, unschedulablePodInfo}
-	if !reflect.DeepEqual(expectedList, q.NominatedPodsForNode("node1")) {
-		t.Error("Unexpected list of nominated Pods for node.")
+	podinfo := q.NominatedPodsForNode("node1")
+	podinfo[0].Pod.Name = "not mpp"
```

Comment:
move this below the L657, and comparing `expectedList` with `podInfos` (instead of call the function again).